PowerEditor/src/functionList.xml

```diff
@@ -90,6 +90,11 @@
 			<association id=  "powershell_function"   langID="53"                          />
 			<association id=  "javascript_function"   langID="58"                          />
 			<association id=       "baanc_section"    langID="60"                          />
+			<!--
+			   As there is currently only one langID for COBOL: change to cobol_section_free
+			   if this is your favourite format
+			-->
+			<association langID="50" id="cobol_section_fixed"/>

```

Comment:
Use same layout as is used for already available associations and position based on `langID` value i.e. between "innosetup_syntax" and "powershell_function" associations.